docs/IE_PLUGIN_DG/PluginTesting.md

```diff
@@ -1,40 +1,60 @@
 # Plugin Testing {#plugin_testing}
 
-Inference Engine (IE) tests infrastructure provides a predefined set of functional tests and utilities exported via the Inference
-Engine developer package. They are used to verify a plugin using the Inference Engine public API.
+Inference Engine (IE) tests infrastructure provides a predefined set of functional tests and utilities. They are used to verify a plugin using the Inference Engine public API.
 All the tests are written in the [Google Test C++ framework](https://github.com/google/googletest).
 
-To build test binaries together with other build artifacts, use the `make all` command. For details, see
-[Build Plugin Using CMake*](@ref plugin_build).
+Inference Engine Plugin tests are included in the `IE::funcSharedTests` CMake target which is built within the  Deep Learning Deployment Toolkit (DLDT) repository
```

Comment:
Is `Deep Learning Deployment Toolkit (DLDT)` is steal valid?

Reply:
I guess we should change it to OpenVINO.

Reply:
fixed